modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -258,6 +259,7 @@ definition {
 
 	test CreateProductBundleWithPriceTypeDynamic {
 		property portal.acceptance = "true";
+		property skip.portal.instance = "CPCommerceProducts#CreateProductBundleWithPriceTypeDynamic";
```

Comment:
Here, and for all the occurences, the right property is:

`property test.name.skip.portal.instance="Testcase#TestName";`

In this particular case there are two properties, just keep:

`property test.name.skip.portal.instance = "CreateProductBundleWithPriceTypeDynamic";`